BraintreeCore/src/test/java/com/braintreepayments/api/BraintreeClientUnitTest.java

```diff
@@ -62,42 +62,20 @@ public void beforeEach() {
 
     @Test
     public void constructor_whenAuthStringIsValidTokenizationKey_shouldNotThrow() {
```

Comment:
I don't really get what these three tests are doing any more. The `BraintreeClient` constructor never throws or returns null, right? So maybe these can be removed? Or changed to do something more useful?

Reply:
Good call - Removed these tests since the valid auth construction is already covered in `AuthorizationUnitTest`